modules/data.land/R/put.veg.module.R

```diff
@@ -49,24 +35,29 @@
     logger.severe(paste(fcn, "does not exist."))
   }
   
+  # NOTE : match_pft is now called in veg2model
```

Comment:
sure

one question though, now that I removed `remove_dead_trees` from upstream veg data will come to `match_pft` with dead tree codes if I take match_pft outside of `veg2model`. If I don't remove them, it won't match with PFTs (in ED's FFT case I think SNAG wont). If I'm going to remove dead trees before match_pft, I have the same problem that not all mode IC want to get rid of dead trees so I'll need checks (e.g. if ED, remove dead trees : but we're trying to avoid hacks I believe, or should I pass a flag via settings and check for that flag?) how should I resolve it?

I don't have that problem if match_pft lives inside veg2model